src/odemis/driver/tlaptmf.py

```diff
@@ -415,6 +415,11 @@ def GetInfo(self):
         values = struct.unpack('<I8sHI48s12xHHH', res)
         sn, modl, typ, fmv, notes, hwv, state, nc = values
 
+        if sys.version_info[0] >= 3:
+            # convert bytes to str
+            modl = modl.decode('latin1')
+            notes = notes.decode('latin1')
+
```

Comment:
Don't worry about doing it all the time. I'd suggest to do it immediately after the rstrip(), like that:
`modl = modl.rstrip("\x00").decode('latin1')`